### PR TITLE
scrum-22 & scrum 23 & scrum-24 & scrum-25

### PR DESCRIPTION
**Pull request by Fenrur**
**As a _MINTER_, I can like/dislike an NFT.**
 - increment/decrement the number of "Mint"s under the NFT
  - send grouped notification to owner
  - grouping should be done with time range of 1h

**As a _MINTER_, I can like/dislike an NFT.**
  - increment/decrement the number of "Mint"s under the NFT
  - send grouped notification to owner
  - grouping should be done with time range of 1h

**As a _MINTER_, I can comment on an NFT.**
 - max 300 characters
  - detect first hand offensive generic words/expressions
  - can be a reply to another comment
  - can mention any username with @ as long as: it's the original poster or someone who posted before me in the same thread of comment (parent comment or same level of nesting)
  - mentions should be limited to 3 max.
  - notify author
  - notify MINTER if it's a reply
  - notify MINTER if there's a mention
  
  **As a _MINTER_, I can load more comments.**
   - by default only loads 20 first comments
  - show "show more comments" button to load more (20 per click)
 
  1902fd9fb87a127b51f340e7d8e78ee84287c68e This is a bad move, I should have deleted it and merged it with another commit

### CODE REVIEW

**Review by fourm1 — APPROVED** [review]
Looks fine

**Review by sckaper — APPROVED** [review]
looks good to me

File: app/api/nft/comment/mint/route.ts [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    switch (result) {
+      case "comment_not_found":
+        return problem(commentNotFoundProblem)
+
+      case "not_minted":
+        return problem(notMintedCommentProblem)
+
+      case "unminted":
+        return NextResponse.json({message: "unminted"})
+    }
```
Comment (avetisk) [inline app/api/nft/comment/mint/route.ts:146]: You should create helper functions for all those checks.

File: app/notifications/follow-request/csr.tsx [hunk 2]
```diff
@@ -360,9 +358,7 @@ export function FollowRequestPageContent() {
         hasMore={sectionIgnoredRequestersMode === "viewing" ? hasMoreIgnoredRequester : false}
         loader={
           <div className="grid justify-center">
-            {
-              <BackgroundLoadingDots size={50}/>
-            }
+            <BackgroundLoadingDots size={50}/>
```
Comment (avetisk) [inline app/notifications/follow-request/csr.tsx:361]: Use a code formatter.

File: src/comment/repository.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    const result = await this.pgClient
+      .select({count: count()})
+      .from(CommentTable)
+      .where(eq(CommentTable.id, commentId))
+
+    return result[0].count > 0
+  }
+
+  public async findComment(profile: number, nftId: number, replyCommentId?: number) {
+    if (replyCommentId) {
```
Comment (avetisk) [inline src/comment/repository.ts:48]: Early return!

And use a formatter!

**Pull request merged**